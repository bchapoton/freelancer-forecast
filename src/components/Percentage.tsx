import React from 'react';

export type PercentageProps = {
    children: number;
};

function Percentage({ children }: PercentageProps) {
    return <span>{children}%</span>;
}

export default Percentage;
