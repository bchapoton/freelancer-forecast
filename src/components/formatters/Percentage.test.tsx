import React from 'react';
import renderer from 'react-test-renderer';
import Percentage from './Percentage';

it('Percentage renders correctly', () => {
    const tree = renderer.create(<Percentage>{80}</Percentage>).toJSON();
    expect(tree).toMatchSnapshot();
});
