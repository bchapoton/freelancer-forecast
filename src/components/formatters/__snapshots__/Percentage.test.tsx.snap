// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Percentage renders correctly 1`] = `
<span>
  80
  %
</span>
`;
